examples/nftvm/tests/integration/nft_collection_test.go

```diff
@@ -0,0 +1,19 @@
+// Tests core logic of NFT Collections and Instances
+
+package integration_test
+
+import "github.com/ava-labs/hypersdk/codec"
+
+const (
+	CollectionNameOne     = "The Louvre Museum"
```

Comment:
Does this need to be a separate file?

Reply:
This was a remanent of the original unit testing setup, I moved it back into `integration_test.go`